### ISSUE TITLE
Can't edit cargo run configurations in CLion master

### ISSUE THREAD

**Issue by lancelote** [intellij-rust/intellij-rust#4888]
## Environment

* **IntelliJ Rust plugin version:** 0.2.115.2840-201-nightly
* **Rust toolchain version:** 1.39.0 (4560ea788 2019-11-04) x86_64-apple-darwin
* **IDE name and version:** CLion 2020.1 Snapshot (CL-201.4520)
* **Operating system:** macOS 10.15

## Problem description

Cargo run configuration is a blank window. Exception:

```
java.lang.NoSuchMethodError: 'com.intellij.ui.layout.CellBuilder com.intellij.ui.layout.Cell.invoke$default(com.intellij.ui.layout.Cell, javax.swing.JComponent, com.intellij.ui.layout.CCFlags[], int, com.intellij.ui.layout.GrowPolicy, java.lang.String, int, java.lang.Object)'
	at org.rust.cargo.runconfig.ui.CargoCommandConfigurationEditor$createEditor$$inlined$panel$lambda$1.invoke(CargoCommandConfigurationEditor.kt:137)
	at org.rust.cargo.runconfig.ui.CargoCommandConfigurationEditor$createEditor$$inlined$panel$lambda$1.invoke(CargoCommandConfigurationEditor.kt:43)
	at org.rust.cargo.runconfig.ui.CargoCommandConfigurationEditor$labeledRow$1.invoke(CargoCommandConfigurationEditor.kt:163)
	at org.rust.cargo.runconfig.ui.CargoCommandConfigurationEditor$labeledRow$1.invoke(CargoCommandConfigurationEditor.kt:43)
	at com.intellij.ui.layout.RowBuilder$DefaultImpls.row(Row.kt:37)
	at com.intellij.ui.layout.Row.row(Row.kt:98)
	at com.intellij.ui.layout.LayoutBuilder.row(LayoutBuilder.kt)
	at com.intellij.ui.layout.RowBuilder$DefaultImpls.row$default(Row.kt:36)
	at org.rust.cargo.runconfig.ui.CargoCommandConfigurationEditor.labeledRow(CargoCommandConfigurationEditor.kt:163)
	at org.rust.cargo.runconfig.ui.CargoCommandConfigurationEditor.createEditor(CargoCommandConfigurationEditor.kt:136)
	at com.intellij.openapi.options.SettingsEditor.getComponent(SettingsEditor.java:94)
	at com.intellij.execution.impl.ConfigurationSettingsEditor$ConfigToSettingsWrapper.createEditor(ConfigurationSettingsEditor.java:294)
	at com.intellij.openapi.options.SettingsEditor.getComponent(SettingsEditor.java:94)
	at com.intellij.openapi.options.GroupSettingsBuilder.doCreateComponent(GroupSettingsBuilder.java:57)
	at com.intellij.openapi.options.GroupSettingsBuilder.createCompoundEditor(GroupSettingsBuilder.java:49)
	at com.intellij.openapi.options.CompositeSettingsEditor.createEditor(CompositeSettingsEditor.java:106)
	at com.intellij.openapi.options.SettingsEditor.getComponent(SettingsEditor.java:94)
	at com.intellij.execution.impl.ConfigurationSettingsEditorWrapper.createEditor(ConfigurationSettingsEditorWrapper.java:97)
	at com.intellij.openapi.options.SettingsEditor.getComponent(SettingsEditor.java:94)
	at com.intellij.openapi.options.SettingsEditorConfigurable.<init>(SettingsEditorConfigurable.java:25)
	at com.intellij.execution.impl.BaseRCSettingsConfigurable.<init>(BaseRCSettingsConfigurable.java:18)
	at com.intellij.execution.impl.SingleConfigurationConfigurable.<init>(SingleConfigurationConfigurable.java:80)
	at com.intellij.execution.impl.SingleConfigurationConfigurable.editSettings(SingleConfigurationConfigurable.java:115)
	at com.intellij.execution.impl.RunConfigurable.getSafeUserObject(RunConfigurable.kt:341)
	at com.intellij.execution.impl.RunConfigurable.access$getSafeUserObject(RunConfigurable.kt:84)
	at com.intellij.execution.impl.RunConfigurable$initTree$4.valueChanged(RunConfigurable.kt:216)
	at java.desktop/javax.swing.JTree.fireValueChanged(JTree.java:2967)
	at java.desktop/javax.swing.JTree$TreeSelectionRedirector.valueChanged(JTree.java:3456)
	at java.desktop/javax.swing.tree.DefaultTreeSelectionModel.fireValueChanged(DefaultTreeSelectionModel.java:641)
	at com.intellij.ui.treeStructure.Tree$MySelectionModel.fireValueChanged(Tree.java:640)
	at java.desktop/javax.swing.tree.DefaultTreeSelectionModel.notifyPathChange(DefaultTreeSelectionModel.java:1111)
	at java.desktop/javax.swing.tree.DefaultTreeSelectionModel.setSelectionPaths(DefaultTreeSelectionModel.java:297)
	at java.desktop/javax.swing.JTree.setSelectionPaths(JTree.java:1669)
	at java.desktop/javax.swing.JTree.setSelectionRows(JTree.java:1752)
	at java.desktop/javax.swing.JTree.setSelectionRow(JTree.java:1727)
	at com.intellij.util.ui.tree.TreeUtil.lambda$showAndSelect$10(TreeUtil.java:615)
	at com.intellij.util.ui.tree.TreeUtil.showAndSelect(TreeUtil.java:650)
	at com.intellij.util.ui.tree.TreeUtil.showAndSelect(TreeUtil.java:570)
	at com.intellij.util.ui.tree.TreeUtil.showAndSelect(TreeUtil.java:565)
	at com.intellij.util.ui.tree.TreeUtil.selectPath(TreeUtil.java:480)
	at com.intellij.util.ui.tree.TreeUtil.selectInTree(TreeUtil.java:999)
	at com.intellij.util.ui.tree.TreeUtil.selectInTree(TreeUtil.java:987)
	at com.intellij.execution.impl.RunConfigurable.selectConfiguration(RunConfigurable.kt:296)
	at com.intellij.execution.impl.RunConfigurable.access$selectConfiguration(RunConfigurable.kt:84)
	at com.intellij.execution.impl.RunConfigurable$selectConfigurableOnShow$1.run(RunConfigurable.kt:274)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:467)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:443)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:420)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:933)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:806)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:433)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:702)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:432)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:117)
	at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:190)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:235)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:233)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:233)
	at java.desktop/java.awt.Dialog.show(Dialog.java:1063)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:708)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:437)
	at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1705)
	at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1664)
	at com.intellij.execution.actions.EditRunConfigurationsAction.actionPerformed(EditRunConfigurationsAction.java:28)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:283)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.java:277)
	at com.intellij.ui.popup.ActionPopupStep.performAction(ActionPopupStep.java:223)
	at com.intellij.ui.popup.ActionPopupStep.performAction(ActionPopupStep.java:213)
	at com.intellij.ui.popup.ActionPopupStep.lambda$onChosen$2(ActionPopupStep.java:207)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:94)
	at com.intellij.ui.popup.AbstractPopup.lambda$dispose$13(AbstractPopup.java:1444)
	at com.intellij.util.ui.UIUtil.invokeLaterIfNeeded(UIUtil.java:2338)
	at com.intellij.ide.IdeEventQueue.ifFocusEventsInTheQueue(IdeEventQueue.java:181)
	at com.intellij.ide.IdeEventQueue.executeWhenAllFocusEventsLeftTheQueue(IdeEventQueue.java:133)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.doWhenFocusSettlesDown(FocusManagerImpl.java:167)
	at com.intellij.openapi.wm.impl.IdeFocusManagerImpl.doWhenFocusSettlesDown(IdeFocusManagerImpl.java:36)
	at com.intellij.ui.popup.AbstractPopup.dispose(AbstractPopup.java:1440)
	at com.intellij.ui.popup.WizardPopup.dispose(WizardPopup.java:152)
	at com.intellij.ui.popup.list.ListPopupImpl.dispose(ListPopupImpl.java:331)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.dispose(PopupFactoryImpl.java:292)
	at com.intellij.openapi.util.ObjectNode.lambda$execute$0(ObjectNode.java:111)
	at com.intellij.openapi.util.ObjectTree.executeActionWithRecursiveGuard(ObjectTree.java:184)
	at com.intellij.openapi.util.ObjectNode.execute(ObjectNode.java:71)
	at com.intellij.openapi.util.ObjectTree.executeAll(ObjectTree.java:134)
	at com.intellij.openapi.util.Disposer.dispose(Disposer.java:116)
	at com.intellij.openapi.util.Disposer.dispose(Disposer.java:106)
	at com.intellij.ui.popup.WizardPopup.disposeAllParents(WizardPopup.java:253)
	at com.intellij.ui.popup.list.ListPopupImpl.handleNextStep(ListPopupImpl.java:466)
	at com.intellij.ui.popup.list.ListPopupImpl._handleSelect(ListPopupImpl.java:420)
	at com.intellij.ui.popup.list.ListPopupImpl.handleSelect(ListPopupImpl.java:366)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.handleSelect(PopupFactoryImpl.java:312)
	at com.intellij.ui.popup.list.ListPopupImpl$MyMouseListener.mouseReleased(ListPopupImpl.java:531)
	at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:298)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6651)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
	at com.intellij.ui.popup.list.ListPopupImpl$MyList.processMouseEvent(ListPopupImpl.java:588)
	at java.desktop/java.awt.Component.processEvent(Component.java:6416)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5026)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2773)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:778)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:751)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:749)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:748)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:933)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:871)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:803)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:433)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:713)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:432)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by tustvold** [comment]
I'm still seeing this issue on the latest plugin and CLion albeit with a slightly different error 

Clion: #CL-201.4865.10, built on February 4, 2020
Intellij-Rust: 0.2.115.2152-201
Operating System: Mac OS 10.14.6 

```
java.lang.IllegalAccessError: class org.rust.cargo.runconfig.ui.CargoCommandConfigurationEditor tried to access private method 'void com.intellij.ui.layout.LayoutKt.initPanel(com.intellij.ui.layout.LayoutBuilder, com.intellij.openapi.ui.DialogPanel)' (org.rust.cargo.runconfig.ui.CargoCommandConfigurationEditor is in unnamed module of loader com.intellij.ide.plugins.cl.PluginClassLoader @193a917c; com.intellij.ui.layout.LayoutKt is in unnamed module of loader com.intellij.util.lang.UrlClassLoader @5ec0a365)
	at org.rust.cargo.runconfig.ui.CargoCommandConfigurationEditor.createEditor(CargoCommandConfigurationEditor.kt:177)
	at com.intellij.openapi.options.SettingsEditor.getComponent(SettingsEditor.java:94)
	at com.intellij.execution.impl.ConfigurationSettingsEditor$ConfigToSettingsWrapper.createEditor(ConfigurationSettingsEditor.java:294)
	at com.intellij.openapi.options.SettingsEditor.getComponent(SettingsEditor.java:94)
	at com.intellij.openapi.options.GroupSettingsBuilder.doCreateComponent(GroupSettingsBuilder.java:57)
	at com.intellij.openapi.options.GroupSettingsBuilder.createCompoundEditor(GroupSettingsBuilder.java:49)
	at com.intellij.openapi.options.CompositeSettingsEditor.createEditor(CompositeSettingsEditor.java:106)
	at com.intellij.openapi.options.SettingsEditor.getComponent(SettingsEditor.java:94)
	at com.intellij.execution.impl.ConfigurationSettingsEditorWrapper.createEditor(ConfigurationSettingsEditorWrapper.java:97)
	at com.intellij.openapi.options.SettingsEditor.getComponent(SettingsEditor.java:94)
	at com.intellij.openapi.options.SettingsEditorConfigurable.<init>(SettingsEditorConfigurable.java:25)
	at com.intellij.execution.impl.BaseRCSettingsConfigurable.<init>(BaseRCSettingsConfigurable.java:18)
	at com.intellij.execution.impl.SingleConfigurationConfigurable.<init>(SingleConfigurationConfigurable.java:80)
	at com.intellij.execution.impl.SingleConfigurationConfigurable.editSettings(SingleConfigurationConfigurable.java:115)
	at com.intellij.execution.impl.RunConfigurable.getSafeUserObject(RunConfigurable.kt:341)
	at com.intellij.execution.impl.RunConfigurable.access$getSafeUserObject(RunConfigurable.kt:84)
	at com.intellij.execution.impl.RunConfigurable$initTree$4.valueChanged(RunConfigurable.kt:216)
	at java.desktop/javax.swing.JTree.fireValueChanged(JTree.java:2967)
	at java.desktop/javax.swing.JTree$TreeSelectionRedirector.valueChanged(JTree.java:3456)
	at java.desktop/javax.swing.tree.DefaultTreeSelectionModel.fireValueChanged(DefaultTreeSelectionModel.java:641)
	at com.intellij.ui.treeStructure.Tree$MySelectionModel.fireValueChanged(Tree.java:640)
	at java.desktop/javax.swing.tree.DefaultTreeSelectionModel.notifyPathChange(DefaultTreeSelectionModel.java:1111)
	at java.desktop/javax.swing.tree.DefaultTreeSelectionModel.setSelectionPaths(DefaultTreeSelectionModel.java:297)
	at java.desktop/javax.swing.JTree.setSelectionPaths(JTree.java:1669)
	at java.desktop/javax.swing.JTree.setSelectionRows(JTree.java:1752)
	at java.desktop/javax.swing.JTree.setSelectionRow(JTree.java:1727)
	at com.intellij.util.ui.tree.TreeUtil.lambda$showAndSelect$10(TreeUtil.java:615)
	at com.intellij.util.ui.tree.TreeUtil.showAndSelect(TreeUtil.java:650)
	at com.intellij.util.ui.tree.TreeUtil.showAndSelect(TreeUtil.java:570)
	at com.intellij.util.ui.tree.TreeUtil.showAndSelect(TreeUtil.java:565)
	at com.intellij.util.ui.tree.TreeUtil.selectPath(TreeUtil.java:480)
	at com.intellij.util.ui.tree.TreeUtil.selectInTree(TreeUtil.java:1013)
	at com.intellij.util.ui.tree.TreeUtil.selectInTree(TreeUtil.java:1001)
	at com.intellij.execution.impl.RunConfigurable.selectConfiguration(RunConfigurable.kt:296)
	at com.intellij.execution.impl.RunConfigurable.access$selectConfiguration(RunConfigurable.kt:84)
	at com.intellij.execution.impl.RunConfigurable$selectConfigurableOnShow$1.run(RunConfigurable.kt:274)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:467)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:443)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:420)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:933)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:806)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:433)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:702)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:432)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:117)
	at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:190)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:235)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:233)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:233)
	at java.desktop/java.awt.Dialog.show(Dialog.java:1063)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:708)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:437)
	at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1705)
	at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1664)
	at com.intellij.execution.actions.EditRunConfigurationsAction.actionPerformed(EditRunConfigurationsAction.java:28)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:283)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.java:277)
	at com.intellij.ui.popup.ActionPopupStep.performAction(ActionPopupStep.java:223)
	at com.intellij.ui.popup.ActionPopupStep.performAction(ActionPopupStep.java:213)
	at com.intellij.ui.popup.ActionPopupStep.lambda$onChosen$2(ActionPopupStep.java:207)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:94)
	at com.intellij.ui.popup.AbstractPopup.lambda$dispose$13(AbstractPopup.java:1454)
	at com.intellij.util.ui.UIUtil.invokeLaterIfNeeded(UIUtil.java:2338)
	at com.intellij.ide.IdeEventQueue.ifFocusEventsInTheQueue(IdeEventQueue.java:181)
	at com.intellij.ide.IdeEventQueue.executeWhenAllFocusEventsLeftTheQueue(IdeEventQueue.java:133)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.doWhenFocusSettlesDown(FocusManagerImpl.java:167)
	at com.intellij.openapi.wm.impl.IdeFocusManagerImpl.doWhenFocusSettlesDown(IdeFocusManagerImpl.java:36)
	at com.intellij.ui.popup.AbstractPopup.dispose(AbstractPopup.java:1450)
	at com.intellij.ui.popup.WizardPopup.dispose(WizardPopup.java:152)
	at com.intellij.ui.popup.list.ListPopupImpl.dispose(ListPopupImpl.java:331)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.dispose(PopupFactoryImpl.java:292)
	at com.intellij.openapi.util.ObjectNode.lambda$execute$0(ObjectNode.java:111)
	at com.intellij.openapi.util.ObjectTree.executeActionWithRecursiveGuard(ObjectTree.java:184)
	at com.intellij.openapi.util.ObjectNode.execute(ObjectNode.java:71)
	at com.intellij.openapi.util.ObjectTree.executeAll(ObjectTree.java:134)
	at com.intellij.openapi.util.Disposer.dispose(Disposer.java:116)
	at com.intellij.openapi.util.Disposer.dispose(Disposer.java:106)
	at com.intellij.ui.popup.WizardPopup.disposeAllParents(WizardPopup.java:253)
	at com.intellij.ui.popup.list.ListPopupImpl.handleNextStep(ListPopupImpl.java:466)
	at com.intellij.ui.popup.list.ListPopupImpl._handleSelect(ListPopupImpl.java:420)
	at com.intellij.ui.popup.list.ListPopupImpl.handleSelect(ListPopupImpl.java:366)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionGroupPopup.handleSelect(PopupFactoryImpl.java:312)
	at com.intellij.ui.popup.list.ListPopupImpl$MyMouseListener.mouseReleased(ListPopupImpl.java:531)
	at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:298)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6651)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
	at com.intellij.ui.popup.list.ListPopupImpl$MyList.processMouseEvent(ListPopupImpl.java:588)
	at java.desktop/java.awt.Component.processEvent(Component.java:6416)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5026)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2773)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:778)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:751)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:749)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:748)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:933)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:871)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:803)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:433)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:713)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:432)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```


I've tried deleting `.idea` and the issue persists.

**Comment by Undin** [comment]
@tustvold It's not the same issue, it's a new one because of new changes in the platform.
Already fixed by #4933. You can wait for the release (should be published next week) or you can try nightly build. See [instruction](https://intellij-rust.github.io/install#nightly-builds) how to install it

**Comment by tustvold** [comment]
@Undin Brilliant, can confirm nightly fixed the issue. Thank you for your rapid response.
